cpp/include/cudf/scalar/scalar.hpp

```diff
@@ -90,7 +90,7 @@ class scalar {
 
   scalar() = delete;
 
-  scalar(scalar&& other) = default;
+  scalar(scalar&& other) = default;  ///< @movedoc{scalar}
```

Comment:
I'm a little unsure of this pattern as well. It hard-codes the parameter name as `other`. I assume this code previously raised a warning for an undocumented method. Is there any other way to remove this warning?

Reply:
Hard-coding can be removed by passing the second argument as parameter to the alias. (will modify alias accordingly).
`@movedoc{scalar, other}`

move constructors are still not supported by doxygen https://github.com/doxygen/doxygen/issues/6249
In fact, most of our doxygen warnings are due to this missing feature.